textproc/OCRmyPDF/Portfile

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        ocrmypdf OCRmyPDF 16.1.2 v
+github.tarball_from releases
+distname            ocrmypdf-${github.version}
+
+categories          textproc
+maintainers         {@akierig fastmail.de:akierig} openmaintainer
+revision            0
+
+checksums           rmd160  25b94fbd304d1fb8235cbd12c1b8fb2af0cf868a \
+                    sha256  d1f9c62b3a0cb090218e8f719d85d110bad95cb73e83dec05842652d173b0c29 \
+                    size    6720680
+
+description         ${name} adds an OCR text layer to scanned PDF files, \
+                    allowing them to be searched
+long_description    ${description}. ${name} OCRmyPDF also supports plugins \
+                    that enable customization of its processing steps, \
+                    and it is highly tolerant of PDFs containing scanned \
+                    images and “born digital” content that doesn’t require text recognition.
+
+supported_archs     noarch
+platforms           {darwin any}
+license             GPL-2
+python.default_version 312
+
+depends_build-append port:py${python.version}-setuptools \
+                     port:py${python.version}-setuptools_scm
+
+depends_lib-append  port:ghostscript \
+                    port:img2pdf \
+                    port:jbig2enc \
+                    port:libpng \
+                    port:pngquant \
+                    port:qpdf \
+                    port:tesseract \
+                    port:unpaper \
+                    port:py${python.version}-cryptography \
+                    port:py${python.version}-deprecated \
```

Comment:
this isn't the same as `py-deprecation` as listed by [upstream](https://github.com/ocrmypdf/OCRmyPDF/blob/a9e1d19b78a9d24bddb07d0c42c8ebb9a1430592/pyproject.toml#L16)

Reply:
So it was throwing an error without py-deprecated but that appears to be related to the fact that `py-pikepdf` needs `py-deprecated` and _not_ `py-depreciation` as is currently in the Portfile. I'll put in a separate PR for that correction. 

cf.: https://github.com/pikepdf/pikepdf/blob/main/pyproject.toml

Reply:
this should be resolved with #23363